content/docs/guides/crosswalk/kubernetes/_index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Pulumi Crosswalk for Kubernetes"
```

Comment:
Can we link to this prominently from https://www.pulumi.com/docs/intro/cloud-providers/kubernetes/? Also worth thinking of ways to improve the visibility of this content -- I still find the AWS Crosswalk content too buried...

Reply:
Done